### DISCUSSION TITLE
Features from HyDE for everyone

### DISCUSSION THREAD

**Discussion by dieBakterie** [prasanthrangan/hyprdots#1349] [General]
Are there plans to make one package or packages that has the Same functionality like the `wallbash.sh`,  `wbarconfgen.sh`, `swww*.sh` Scripts?

**Comment by ghost** [comment]
Well, it is pretty easy to just take the script and apply it somewhere else, and you're completely allowed to do so.

The only thing i know that there has been discussion of separating from HyDE is wallbash, but no promises on that.
